radiance_folder/model/README.md

```diff
@@ -0,0 +1,127 @@
+simulation the `.blk` files will not be used.
+
+```shell
+oconv ./model/scene/*.mat ./model/scene/*.rad \
+  ./model/aperture/*.mat ./model/aperture/*.rad > static_scene.oct
+```
+
+In `dynamic` models the `static apertures` should be separated from the rest of the the
+geometries. This is required for blacking-out the static aperture during the isolated
+contribution calculation.
+
+```shell
+oconv ./model/scene/*.mat ./model/scene/*.rad > static_scene_no_aperture.oct
+```
+
+For simulating the contribution of each isolated aperture the static_scene octree should
+be mixed with the isolated aperture while the rest of apertures are turned black to block
+the light entrance. For instance the octree for calculating the isolated contribution
+from the static apertures will look like this:
+
+```shell
+oconv -i static_scene_no_aperture.oct \
+  ./model/aperture/*.mat ./model/aperture/*.rad \
+  ./model/aperture_group/*.blk ./model/aperture_group/*.rad \
+  ... \
+  > static_scene.oct
+```
+
+The number of total runs will be:
+
+((state count for all aperture groups) + 1) * state count for all dynamic geometry
+
+For our case which only has one single aperture group with two states this number will be
+`(2 + 1) * 1`.
+
+
+## Direct sunlight
+
+The main difference between direct sunlight studies and ray-tracing with diffuse and
+reflected light is that the all non-aperture geometry in the scene must be black.
+
+```shell
+oconv ./model/scene/*.blk ./model/scene/*.rad \
+  ./model/aperture/*.mat ./model/aperture/*.rad > static_scene.oct
+```
+
+The same stands for modeling `dynamic` models. The workflow for calculating the
+contribution for dynamic models is similar to `raytracing` except that the non-aperture
+must be black.
+
+## View matrix
+
+View matrix calculation is similar to `raytracing`.
+
+
+## Daylight matrix
+
+View matrix calculation is similar to `raytracing`.
+
+
+
+## 2-Phase
+
+2 phase is the combination of `ray-tracing` and `direct sunlight` calculation
+followed by matrix multiplication. 
+
+## 3-Phase
+
+3 phase is the combination of `view matrix` and `daylight matrix` calculation
+followed by matrix multiplication. 
+
+## 5-Phase
+
+5 phase is the combination of `view matrix`, `daylight matrix` and `direct sunlight`
+calculation followed by matrix multiplication.
+
+
+# Modeling inter-connected spaces, double wall or light-pipes with aperture groups
+
+For inter-connected spaces another layer of complexity is added to the calculation if one
+needs to use 3 or 5 phase methods which needs extra coefficient calculation between
+senders and receivers. However for `raytracing`, `direct sunlight` and `2-phase`
+simulations you can use the folder structure as is. The only consideration is to ensure
+to model the nonopaque geometry which connects the spaces as an _interior_ aperture.
+
+Each sensor grid must provide the list of apertures that should be included in the study
```

Comment:
Is this a `must` or is it optional but recommended?

Reply:
For case of interconnected spaces it is highly recommended.